x-pack/plugins/security/public/mocks.ts

```diff
@@ -10,13 +10,11 @@ import type { MockAuthenticatedUserProps } from '../common/model/authenticated_u
 import { mockAuthenticatedUser } from '../common/model/authenticated_user.mock';
 import { authenticationMock } from './authentication/index.mock';
 import { navControlServiceMock } from './nav_control/index.mock';
-import { createSessionTimeoutMock } from './session/session_timeout.mock';
 import { getUiApiMock } from './ui_api/index.mock';
 
 function createSetupMock() {
   return {
     authc: authenticationMock.createSetup(),
-    sessionTimeout: createSessionTimeoutMock(),
```

Comment:
I noticed this was dead code so I removed it, along with `session_timeout.mock.ts`